armi/reactor/systemLayoutInput.py

```diff
@@ -92,7 +92,7 @@
                         vol.Optional(
                             INP_SYMMETRY,
                             default=geometry.THIRD_CORE + geometry.PERIODIC,
-                        ): vol.In(geometry.VALID_SYMMETRY),
+                        ): vol.In(geometry.SymmetryType.validSymmetryStrings()),
```

Comment:
As mentioned above, probably should provide this function at all. This schema can do the set comprehension itself